operations/ntl/cc/kernels/ntl_matrix_ops.cc

```diff
@@ -138,6 +138,10 @@ class MatMulNTLOp : public OpKernel {
     const NTLMatrix * m2 = nullptr;
     OP_REQUIRES_OK(ctx, GetNTLMatrix(ctx, 1, &m2));
 
+    const Tensor& mod = ctx->input(2);
```

Comment:
I'm wondering if this shouldn't be taken from one of the other two tensors instead of being given explicitly. If explicit then I see users grapping it from one of those anyways, and it would be a fault to pass in anything else.